data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderAppConfig.java

```diff
@@ -59,4 +65,41 @@ public PeerForwarder peerForwarder(final PeerForwarderClientFactory peerForwarde
         return new PeerForwarder(peerForwarderClientFactory, peerForwarderClient);
     }
 
+    @Bean
+    public RequestExceptionHandler requestExceptionHandler() {
+        return new RequestExceptionHandler();
+    }
+
+    @Bean
+    public PeerForwarderHttpService peerForwarderHttpService(
+            final RequestExceptionHandler requestExceptionHandler,
+            final ObjectMapper objectMapper
+    ) {
+        return new PeerForwarderHttpService(requestExceptionHandler, objectMapper);
+    }
+
+    @Bean
+    public PeerForwarderHttpServerProvider peerForwarderHttpServerProvider(
+            final PeerForwarderConfiguration peerForwarderConfiguration,
+            final CertificateProviderFactory certificateProviderFactory,
+            final PeerForwarderHttpService peerForwarderHttpService
+    ) {
+        return new PeerForwarderHttpServerProvider(peerForwarderConfiguration,
+                certificateProviderFactory, peerForwarderHttpService);
+    }
+
+    @Bean(name="armeria_server")
```

Comment:
Perhaps rename this to `peerForwarderServer`. Two reasons:
* The name is useful to disambiguate it. Data Prepper probably won't have more than one peer-forwarder server, but may have multiple Armeria servers.
* I think we are using camelCase elsewhere